dashboard/tests/test_dashboard.py

```diff
@@ -442,5 +444,53 @@ def test_aiohttp_cache(enable_test_module, ray_start_with_dashboard):
     assert len(data[0]) == 2
 
 
+def test_get_cluster_status(ray_start_with_dashboard):
+    assert (wait_until_server_available(ray_start_with_dashboard["webui_url"])
+            is True)
+    address_info = ray_start_with_dashboard
+    webui_url = address_info["webui_url"]
+    webui_url = format_web_url(webui_url)
+
+    # Check that the cluster_status endpoint works without the underlying data
+    # from the GCS, but returns nothing.
+    for _ in range(5):
+        time.sleep(1)
+        try:
+            response = requests.get(f"{webui_url}/api/cluster_status")
+            response.raise_for_status()
+            assert response.json()["result"]
+            assert "autoscalingStatus" in response.json()["data"]
+            assert response.json()["data"]["autoscalingStatus"] is None
+            assert "autoscalingError" in response.json()["data"]
+            assert response.json()["data"]["autoscalingError"] is None
+            break
+        except requests.RequestException:
```

Comment:
It'll be easier to debug a failure if you set a var `last_exc` here and raise that in the else case of the for loop instead.

Reply:
good idea